src/step-input.scss

```diff
@@ -0,0 +1,115 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-step-input;
+
+.#{$block} {
+  $fd-button-icon-font-size: 0.875rem;
+  $fd-step-input-cozy-height: 2.25rem;
+  $fd-step-input-compact-height: 1.5rem;
+  $fd-step-input-cozy-button-width: 2.125rem;
+  $fd-step-input-compact-button-width: 1.5rem;
+  $fd-step-input-description-padding: 0.5rem;
+
+  @mixin iconSize($size) {
+    &::before,
+    &::after {
+      font-size: $size;
+      margin: 0;
+    }
+  }
+
+  @include fd-reset();
+  @include fd-form-states();
+  @include fd-fiori-focus-within();
+
+  width: auto;
+  border-style: solid;
+  border-width: var(--sapField_BorderWidth);
+  border-color: var(--sapField_BorderColor);
+  border-radius: var(--sapField_BorderCornerRadius);
+  background-color: var(--sapField_Background);
+  height: calc(#{$fd-step-input-cozy-height} + 2 * var(--sapField_BorderWidth));
```

Comment:
I don't know if this is correct because there is no exact value in spec, but our standard input height is `36px` and step-input height is `38px`.

Reply:
You are right, there should be `36px` with borders included (34px + 2px)